Library/ViewModels/PledgeAmountViewModel.swift

```diff
@@ -18,7 +18,7 @@ public protocol PledgeAmountViewModelInputs {
 }
 
 public protocol PledgeAmountViewModelOutputs {
-  var amount: Signal<(Double, Bool), Never> { get }
+  var amount: Signal<(Double, Double, Double, Bool), Never> { get }
```

Comment:
Could we just change the type of this output to `Signal<PledgeAmountData, Never>`?